include/class.format.php

```diff
@@ -529,6 +529,10 @@ function __formatDate($timestamp, $format, $fromDb, $dayType, $timeType,
         if (!$timestamp || !($datetime = DateTime::createFromFormat('U', $timestamp)))
             return '';
 
+        // Normalize timezone
+        if ($timezone)
+            $timezone = Format::timezone($timezone);
```

Comment:
Where were the timezones invalid before? From what I tested the timezone showed the same before and after this code.

Reply:
Parsing a timezone from a datetime from database you can end up with a timezone like  'GMT' or '+00:00' - which are not valid. 

Now that you mentioned it, it looks line 538 does the same thing - will drop this change.